docs/command_line_tool/phylum_parse.md

```diff
@@ -4,16 +4,31 @@ category: 6255e67693d5200013b1fa3e
 hidden: false
 ---
 
-Parse a lockfile and output the packages as JSON
```

Comment:
Another lost detail that might be worth included in the command definition.

Reply:
Decided to expand the about section of the Command in general.